Modificación de Login

Andoria-Coss opened the pull request:
Se modificó el Login, forgot-password, y auth (para el aside y el footer). También se agregaron los archivos para el logo (tecnm.png), y para el aside (ceppi.png y ceppia.png).

Andoria-Coss committed:
Modificación de Login

Se modificó el Login, forgot-password, y auth (para el aside y el footer). También se agregaron los archivos para el logo (tecnm.png), y para el aside (ceppi.png y ceppia.png).

Braulio-Palagot committed:
FIX: Ejemplo de Internacionalización de textos.

Braulio-Palagot commented on src/app/modules/auth/components/forgot-password/forgot-password.component.html:
```diff
@@ -3,29 +3,29 @@
     <div class="text-gray-500 fw-semibold fs-6">
-        Enter your email to reset your password.
+        Por favor ingresa tu correo para reestablecer tu contraseña.
     </div>
```
Por favor, utiliza la internacionalización de textos, toma de ejemplo el botón de "¿Olvidaste tu contraseña?" que te subí aquí mismo en tu rama.

Aplica para todos los textos, por favor.

Braulio-Palagot commented on src/app/modules/auth/components/forgot-password/forgot-password.component.html:
```diff
@@ -34,103 +34,124 @@ <h1 class="text-gray-900 fw-bolder mb-3">
+      Cancelar
     </a>
   </div>
   <!--end::Form group-->
```
El botón de "Cancelar" debe ser diferente al de enviar, para cumplir con las reglas de UI/UX de botones primarios y secundarios.

El botón primario es el de "Enviar" y el secundario es el de "Cancelar". Como sugenrencia, el secundario pueder un Outlined-Button.

Braulio-Palagot reviewed (changes requested):
Por favor, atiende los comentarios y avisame cuando esté listo.

Andoria-Coss committed:
Internacionalización y corrección de botón "Cancelar"

Se realizó la internacionalización de los textos y la corrección del color del botón "Cancelar", se editaron los archivos: forgot-password.component.html, forgot-password.component.scss, login.component.html, auth.component.html y es.ts.

Braulio-Palagot committed:
Merge branch 'feature-login' of https://github.com/Braulio-Palagot/sistema-propiedad_intelectual into feature-login

# Conflicts:
#	src/app/modules/auth/components/forgot-password/forgot-password.component.html
#	src/app/modules/i18n/vocabs/es.ts
